source/particle/property/crystal_preferred_orientation.cc

```diff
@@ -1003,6 +1054,32 @@ namespace aspect
               ref_resolved_shear_stress[3] = 1;
               break;
 
+            // RRSS for CPX (preliminary results, require further investigations)
+            // Main slip systems from Bascou_etal., 2002 JSG and Zhang et al., 2006 EPSL and with numerical experiments
+            // case DeformationType::clinopyroxene :
+            //   ref_resolved_shear_stress[0] = 1;
+            //   ref_resolved_shear_stress[1] = 5;
+            //   ref_resolved_shear_stress[2] = 5;
+            //   ref_resolved_shear_stress[3] = 1.5;
+            //   break;
+
+            case DeformationType::clinopyroxene:
+              // Use the member variable CPX_RRSS if it has been set (size == 4)
+              if (CPX_RRSS.size() == 4)
+                {
+                  for (unsigned int i=0; i<4; ++i)
+                    ref_resolved_shear_stress[i] = CPX_RRSS[i];
+                }
+              else
+                {
+                  // Default values if CPX_RRSS wasn't set
+                  ref_resolved_shear_stress[0] = 1;
+                  ref_resolved_shear_stress[1] = 5;
+                  ref_resolved_shear_stress[2] = 5;
+                  ref_resolved_shear_stress[3] = 1.5;
+                }
```

Comment:
This is not wrong, but these are the default values you specify for the input parameter as well. You can just do the following and will get the same results no matter whether you set or not set the values in the input file:
```suggestion
            case DeformationType::clinopyroxene:
                  for (unsigned int i=0; i<4; ++i)
                    ref_resolved_shear_stress[i] = CPX_RRSS[i];
```